.rubocop.yml

```diff
@@ -1,14 +1,46 @@
- Style/FrozenStringLiteralComment:
-   Enabled: false
- 
- Style/StringLiterals:
+ClassLength:
+  Max: 100
+
+Documentation:
+  Enabled: false
+
+Style/FrozenStringLiteralComment:
+  Enabled: false
+
+Style/StringLiterals:
    EnforcedStyle: double_quotes
 
-AllCops:
-  Include:
-    - Rakefile
-    - config.ru
-    - lib/**/*.rake
-  Exclude:
-    - db/schema.rb
- 
+Style/MixinUsage:
+  Enabled: false
+
+Style/CollectionMethods:
+  PreferredMethods:
+    collect: "map"
+    collect!: "map!"
+    inject: "reduce"
+    detect: "find"
+    find_all: "select"
+
+Layout/TrailingBlankLines:
+  Enabled: false
+
+Layout/ClassStructure:
+  Categories:
+    module_inclusion:
+      - include
+      - prepend
+      - extend
+  ExpectedOrder:
+      - module_inclusion
+      - constants
+      - public_class_methods
+      - initializer
+      - public_methods
+      - protected_methods
+      - private_methods
+
+Metrics/MethodLength:
+  Max: 8
+
+Metrics/LineLength:
+  Enabled: false
```

Comment:
We would max length to be 80 characters.
Maybe exclude specific files if this rule is giving you trouble?

Reply:
Okay :)
I will add 80 back in and remove the offending files rather than break the lines?

Reply:
You can add a rule to exclude specific files or directories.
Alternatively, you can fix the violations.
Whichever you choose :)

Reply:
I can't seem to use
```AllCops:
    Exclude:
      - "config/environments/production.rb"
```

I've spent a lot of time reading the docs and various posts and theoretically this should exclude the file right? It's not. I go and fix the violations by breaking the comments across lines and then I get "trailing whitespace" violations and there isn't any trailing whitespace!

Should I not be excluding the file in the way that I'm doing it?

Reply:
I did it by excluding files under the specific cop rather than excluding globally.